low-level/files/opt/seagate/sspl/setup/sspl_test.py

```diff
@@ -56,14 +57,35 @@ def validate(self):
             "cortx-sspl-test": None
             }
         # python 3rd party package dependency
-        pip3_3ps_packages_test = {
-            "Flask": "1.1.1"
+        pip3_3ps_packages_dep = {
+            "Flask": "1.1.1",
+            "coverage": "5.5"
             }
+        # Collect installed python pkg from dependency list.
+        installed_pip3_pkgs = {}
+        for pkg in pkg_resources.working_set:
+            if pkg.project_name in pip3_3ps_packages_dep:
+                installed_pip3_pkgs[pkg.project_name] = pkg.version
+        # Validate pip3 python pkg with required version.
+        for pkg, version in pip3_3ps_packages_dep.items():
+            if pkg == "coverage" and not self.coverage_enabled:
```

Comment:
use this condition to above before line 64 to remove coverage from pip3_3ps_packages_dep  dict itself, so in for loop we dont deal with any specifics

Reply:
addressed. 👍